src/FsCodec.NewtonsoftJson/Codec.fs

```diff
@@ -119,14 +135,16 @@ type Codec private () =
             /// Configuration to be used by the underlying <c>Newtonsoft.Json</c> Serializer when encoding/decoding. Defaults to same as <c>Settings.Create()</c>
             [<Optional; DefaultParameterValue(null)>] ?settings,
             /// Enables one to fail encoder generation if union contains nullary cases. Defaults to <c>false</c>, i.e. permitting them
-            [<Optional; DefaultParameterValue(null)>] ?rejectNullaryCases)
+            [<Optional; DefaultParameterValue(null)>] ?rejectNullaryCases,
+            /// Enables unions to contain a Guid or most primitives. Defaults to <c>true</c>, i.e. preventing Guids and primitives
```

Comment:
`require` does not _Enables_ ....;
/// Controls whether bodies of union cases are permitted to use primitive types, which can be harder to evolve when compared to always wrapping in a record in order to force all fields to have names. Defaults to <c>true</c>, i.e. preventing Guids and primitives from being used directly